aiohttp/helpers.py

```diff
@@ -1038,3 +1039,19 @@ def parse_http_date(date_str: Optional[str]) -> Optional[datetime.datetime]:
             with suppress(ValueError):
                 return datetime.datetime(*timetuple[:6], tzinfo=datetime.timezone.utc)
     return None
+
+
+# Addresses can be either tuples of varying lengths (AF_INET, AF_INET6,
+# AF_NETLINK, AF_TIPC) or strings (AF_UNIX).
```

Comment:
Looks like this could be a part of the docstring. The use case for code comments is to explain motivation. The use case for docstrings is to explain implementation/expectations — treat these like docs for the developers, as they will also show up in REPL and code editors in places where the function is used.